lib/tasks/users/get_an_identity_syncing.rake

```diff
@@ -21,7 +21,7 @@ namespace :users do
                   .with_get_an_identity_id # Must have a get_an_identity_id
                   .where(get_an_identity_id_synced_to_ecf: false) # Must not have already been synced
 
-      users.each do |user|
+      users.find_each(batch_size: 200) do |user|
```

Comment:
Reduced from the default 1000 to be certain we don't run into memory issues, this specific number isn't particularly meaningful we just wanted to be careful